src/UI/ledger/LedgerLoginContainer/index.tsx

```diff
@@ -52,18 +53,25 @@ export const LedgerLoginContainer = ({
     }
   ] = useLedgerLogin({ callbackRoute, token, onLoginRedirect, nativeAuth });
 
-  function getContent() {
+  const getContent = () => {
     if (isLoading) {
       return (
-        <PageState
-          icon={faCircleNotch}
-          iconClass={classes.spinner}
-          title={ledgerWaitingText}
+        <LedgerLoading
+          customSpinnerComponent={customSpinnerComponent}
+          innerLedgerComponentsClassNames={innerLedgerComponentsClassNames}
+          customContentComponent={customContentComponent}
         />
       );
     }
+
     if (ledgerAccount != null && !error) {
-      return <ConfirmAddress token={token} />;
+      return (
+        <ConfirmAddress
+          token={token}
+          innerLedgerComponentsClassNames={innerLedgerComponentsClassNames}
```

Comment:
`customContentComponentClassName`

Reply:
Let's keep it like this because it might get confused with the custom component you can insert, which is unrelated.

Reply:
Changed. 👍